src/melee/ft/ftcoll.c

```diff
@@ -188,3 +170,6882 @@ f32 func_800765F0(Fighter* fp, HSD_GObj* victim, f32 unk_floatvar) // Unk knockb
+/* 8007AC6C 0007784C  41 82 00 28 */	beq lbl_8007AC94
+/* 8007AC70 00077850  80 8D AE B4 */	lwz r4, p_ftCommonData(r13)
+/* 8007AC74 00077854  80 04 07 E8 */	lwz r0, 0x7e8(r4)
+/* 8007AC78 00077858  7C 00 18 40 */	cmplw r0, r3
+/* 8007AC7C 0007785C  41 81 00 18 */	bgt lbl_8007AC94
+/* 8007AC80 00077860  80 04 07 EC */	lwz r0, 0x7ec(r4)
+/* 8007AC84 00077864  7C 03 00 40 */	cmplw r3, r0
+/* 8007AC88 00077868  41 81 00 0C */	bgt lbl_8007AC94
+/* 8007AC8C 0007786C  38 60 00 01 */	li r3, 1
+/* 8007AC90 00077870  4E 80 00 20 */	blr 
+lbl_8007AC94:
+/* 8007AC94 00077874  38 60 00 00 */	li r3, 0
+/* 8007AC98 00077878  4E 80 00 20 */	blr 
+} // clang-format on
+#pragma pop
+
+extern unk_t func_80037ECC();
+
+#pragma push
+asm unk_t func_8007AC9C()
+{ // clang-format off
+    nofralloc
+/* 8007AC9C 0007787C  7C 08 02 A6 */	mflr r0
+/* 8007ACA0 00077880  28 04 01 69 */	cmplwi r4, 0x169
+/* 8007ACA4 00077884  90 01 00 04 */	stw r0, 4(r1)
+/* 8007ACA8 00077888  94 21 FF F8 */	stwu r1, -8(r1)
+/* 8007ACAC 0007788C  80 C5 00 2C */	lwz r6, 0x2c(r5)
+/* 8007ACB0 00077890  90 83 00 20 */	stw r4, 0x20(r3)
+/* 8007ACB4 00077894  41 82 00 28 */	beq lbl_8007ACDC
+/* 8007ACB8 00077898  80 6D AE B4 */	lwz r3, p_ftCommonData(r13)
+/* 8007ACBC 0007789C  80 03 07 E8 */	lwz r0, 0x7e8(r3)
+/* 8007ACC0 000778A0  7C 00 20 40 */	cmplw r0, r4
+/* 8007ACC4 000778A4  41 81 00 18 */	bgt lbl_8007ACDC
+/* 8007ACC8 000778A8  80 03 07 EC */	lwz r0, 0x7ec(r3)
+/* 8007ACCC 000778AC  7C 04 00 40 */	cmplw r4, r0
+/* 8007ACD0 000778B0  41 81 00 0C */	bgt lbl_8007ACDC
+/* 8007ACD4 000778B4  38 00 00 01 */	li r0, 1
+/* 8007ACD8 000778B8  48 00 00 08 */	b lbl_8007ACE0
+lbl_8007ACDC:
+/* 8007ACDC 000778BC  38 00 00 00 */	li r0, 0
+lbl_8007ACE0:
+/* 8007ACE0 000778C0  2C 00 00 00 */	cmpwi r0, 0
+/* 8007ACE4 000778C4  41 82 00 24 */	beq lbl_8007AD08
+/* 8007ACE8 000778C8  88 66 20 84 */	lbz r3, 0x2084(r6)
+/* 8007ACEC 000778CC  54 60 CF FF */	rlwinm. r0, r3, 0x19, 0x1f, 0x1f
+/* 8007ACF0 000778D0  40 82 00 18 */	bne lbl_8007AD08
+/* 8007ACF4 000778D4  38 00 00 01 */	li r0, 1
+/* 8007ACF8 000778D8  50 03 3E 30 */	rlwimi r3, r0, 7, 0x18, 0x18
+/* 8007ACFC 000778DC  98 66 20 84 */	stb r3, 0x2084(r6)
+/* 8007AD00 000778E0  7C A3 2B 78 */	mr r3, r5
+/* 8007AD04 000778E4  4B FB D1 C9 */	bl func_80037ECC
+lbl_8007AD08:
+/* 8007AD08 000778E8  80 01 00 0C */	lwz r0, 0xc(r1)
+/* 8007AD0C 000778EC  38 21 00 08 */	addi r1, r1, 8
+/* 8007AD10 000778F0  7C 08 03 A6 */	mtlr r0
+/* 8007AD14 000778F4  4E 80 00 20 */	blr 
+} // clang-format on
+#pragma pop
+
+#pragma push
+asm unk_t func_8007AD18()
+{ // clang-format off
+    nofralloc
+/* 8007AD18 000778F8  7C 08 02 A6 */	mflr r0
+/* 8007AD1C 000778FC  90 01 00 04 */	stw r0, 4(r1)
+/* 8007AD20 00077900  94 21 FF D8 */	stwu r1, -0x28(r1)
+/* 8007AD24 00077904  93 E1 00 24 */	stw r31, 0x24(r1)
+/* 8007AD28 00077908  7C 9F 23 78 */	mr r31, r4
+/* 8007AD2C 0007790C  80 04 00 00 */	lwz r0, 0(r4)
+/* 8007AD30 00077910  2C 00 00 02 */	cmpwi r0, 2
+/* 80
```

Comment:
we should make sure to paste in the decomp URLs here @ribbanya .
https://decomp.me/scratch/mL8R8 here's this one, but there are also more that are missing